pandas/_libs/tslibs/period.pyx

```diff
@@ -336,7 +407,7 @@ cdef int dInfoCalc_SetFromAbsTime(date_info *dinfo, double abstime) nogil:
     dinfo.hour = hour
     dinfo.minute = minute
     dinfo.second = second
-    return 0
```

Comment:
why do you need a return?

Reply:
I don't.  Will remove when killing this function off.

Reply:
just remove the return